src/packages/ratmama/ratmama_parser.py

```diff
@@ -108,6 +109,16 @@ async def handle_ratmama_announcement(ctx: Context) -> None:
     lang_code: str = result.group("language_code")
     nickname: Optional[str] = result.group("nick")
 
+    client = await User.from_pydle(ctx.bot, client_name)
+
+    if client is not None:
+        client_hostname = client.hostname
```

Comment:
[Error] Redundant variable assignment

You needlessly assign an instance attribute of a dataclass to a local name (read only, cheap retrieval) , it is better NOT to extract that one attribute from the object, as it may lead to broken expectations.